.github/workflows/main.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     if: "!contains(github.event.head_commit.message, 'skip ci')"
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [14.18.0]
```

Comment:
Mayne this should be automated?

https://github.com/actions/setup-node/issues/32#issuecomment-525791142

```yml
- name: Read .nvmrc
  run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
  id: nvm

- name: Use Node.js (.nvmrc)
  uses: actions/setup-node@v1
  with:
    node-version: "${{ steps.nvm.outputs.NVMRC }}"
```

Reply:
🚀